app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -0,0 +1,50 @@
+module Search
+  # TODO[@atsmith813]: Rename this to CommentSerializer once Elasticsearch is removed
+  class PostgresCommentSerializer < ApplicationSerializer
+    attribute :id, &:search_id
+
+    attributes :path, :public_reactions_count
+
+    attribute :body_text, &:body_markdown
+
+    attribute :class_name do |comment|
+      comment.class.name
+    end
```

Comment:
Yep! I was thinking if we ever decide to simply/change how comments work and the model and/or table was renamed to something like `ArticleComment` this would just work ™️. Unlikely I supposed. I changed it to `Comment`.